dll/cpl/sysdm/general.c

```diff
@@ -498,6 +498,45 @@ static VOID GetSystemInformation(HWND hwnd)
     }
 }
 
+static VOID GetSystemVersion(HWND hwnd)
+{
+    HWND hRosVersion;
+    SIZE_T lenStr, lenVersion, lenDest;
+    LPWSTR szVersion;
+    PVOID szStr, szDest;
+
+    lenVersion = LoadString(hApplet, IDS_ROSVERSION, (LPWSTR)&szVersion, 0);
```

Comment:
This is not going to change by language, so you might as well just use `REACTOS_STR_PRODUCT_VERSION` in the code.

Reply:
I've tried that, it shows an error:

`../dll/cpl/sysdm/general.c:509:38: error: 'REACTOS_STR_PRODUCT_VERSION' undeclared (first use in this function)`